README.md

```diff
@@ -11,54 +11,437 @@
+
+### Rest Endpoint Summary
+
+- /wallets
+- /wallets/{walletId}
+- /wallets/{walletId}/tip
+- /wallets/{walletId}/statistics/utxos
+- /wallets/{walletId}/passphrase
+- /wallets/{walletId}/transactions
+- /wallets/{walletId}/payment-fees
+- /wallets/{walletId}/transactions/{transactionId}
+- /wallets/{walletId}/addresses
+- /stake-pools
+- /stake-pools/{stakePoolId}/wallets/{walletId}
+- /stake-pools/\*/wallets/{walletId}
+- /wallets/{walletId}/delegation-fees
+- /wallets/{walletId}/coin-selections/random
+- /byron-wallets
+- /byron-wallets/{walletId}
+- /byron-wallets/{walletId}/passphrase
+- /byron-wallets/{walletId}/tip
+- /byron-wallets/{walletId}/addresses
+- /byron-wallets/{walletId}/transactions
+- /byron-wallets/{walletId}/payment-fees
+- /byron-wallets/{walletId}/transactions/{transactionId}
+- /byron-wallets/{walletId}/migrations
+- /byron-wallets/{sourceWalletId}/migrations/{targetWalletId}
+- /byron-wallets/{walletId}/statistics/utxos
+- /network/information
+- /network/clock
+- /network/parameters/{epochId}
+- /proxy/transactions
+
+### CLI Summary
+
+```console
+Usage: cardano-wallet COMMAND
+  Cardano Wallet Command-Line Interface (CLI)
+
+Available OPTIONS:
+  -h | --help
+
+Available COMMANDS:
+  launch                Launch and monitor a wallet server and its chain producers.
+  serve                 Serve an HTTP API that listens for commands/actions.
+  mnemonic
+    generate            Generate BIP-39 mnemonic words
+    reward-credentials  Derive reward account private key from mnemonic.
+  wallet
+    list                List all known wallets
+    create
+      from-mnemonic     Create a new wallet using a mnemonic
+      from-public-key   Create a wallet using a public account key
+    get                 Fetch a particular wallet
+    utxo                Get a wallet's UTxO distribution
+    update
+      passphrase        Update a wallet's master passphrase
+      name              Update a wallet's name
+    delete              Forget a wallet and its metadata
+  transaction
+    create              Create a transaction from a known wallet
+    fees                Estimate fees for a transaction
+    list                List the transactions associated with a wallet
+    submit              Submit an externally-signed transaction
+    forget              Forget a pending transaction with specified id
+  address
+    list                List all known addresses of a wallet
+  stake-pool
+      list              List all known stake pools
+  network
+    information         View network information
+    parameters          View network parameters
+    clock               View NTP offset
+  key
+    root                Extract root extended private key from a mnemonic sentence.
+    child               Derive child keys.
+    public              Extract public key from a private key.
+    inspect             Show information about a key.
+  version               Show the program's current version
+```
```

Comment:
This will also not be maintainable. Especially because we already have a CLI manual that is painful enough to maintain. At least the manual provide extra information on top of the command-line help. But copy/pasting the output of `--help` has no value and simply make the README longer.